ApplicationTemplates/rest/handlers/BaseHandler.cfc

```diff
@@ -74,7 +74,7 @@ component extends="coldbox.system.EventHandler"{
 		
 		// Global Response Headers
 		prc.response.addHeader( "x-response-time", prc.response.getResponseTime() )
-				.addHeader( "x-cached-response", prc.response.getCachedResponse() )
+				.addHeader( "x-cached-response", prc.response.getCachedResponse() );
```

Comment:
This is where your change went. You didn't create a branch for each pull request so all your commits to development ended up in this pull.  Since I'm closing this pull, you need to revert your changes in your repo, create a branch, add this semi-colon change back to there and resubmit a pull.  Git only allows you to have one pull request per branch.